pages/studentPortal/intro/Javascript/Find-Plant-video.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import ProgressBar from "../../../../components/coding/studentPortal/ProgressBar";
+import { Button } from "../../../../components/ui/Button";
+
+const JS_FindPlant = () => {
+  return (
+    <>
+      <div className="m-4 mx-12">
+        <ProgressBar completed={100} />
+      </div>
+
+      <div className="mx-12">
+        <iframe
```

Comment:
this is fine as is, but we could add this iframe into the LessonComponent and define a video lesson component in the if statement.